### DISCUSSION TITLE
Add instrument icon for Band Director/Conductor

### DISCUSSION THREAD

**Discussion by henkdegroot** [jamulussoftware/jamulus#1296] [Ideas]
As these days Jamulus is used a lot for band/choir practice....it seems it lacks a proper icon for the director/leader.
I know in most bands they like to play/sing-along anyway but it would be nice if these special people can get their own icon.

**Comment by hoffie** [comment]
Thanks for bringing this up. :)
Hasn't been brought up in our rehearsals, but this sounds logical. I wonder where a good place to track this would be. There is https://github.com/jamulussoftware/jamulus/issues/899 for creating a new instrument icon set, so maybe there?

**Reply by henkdegroot** [reply]
I thought about reporting that in #899 but as this icon/instrument does not exists yet, I thought about just posting in idea catergory. Think this is the first time that I name our band director an instrument :)

**Comment by dakhubgit** [comment]
I'd suggest that we need 5 different icons for a band director, and the director can use keys 1, 2, 3, 4, space to switch between them.

And in case the point of this suggestion is not clear: the icons would be a hand with a baton pointing up, left, right, down.  And maybe straight ahead?  That would enable directors to, well, direct.
